### PR TITLE
usage: Update scripting documentation for ciao

### PR DESCRIPTION
**Pull request by rbradford**
Update the scripting documentation to reflect the change from ciao-cli
to ciao.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

File: ciao/usage/scripting.md [hunk 1]
```diff
@@ -4,29 +4,30 @@ permalink: scripting.html
 keywords: CIAO, command line, tool, scripting, templates
 ---
 
-## Scripting with ciao-cli
+## Scripting with ciao
 
-Most of the ciao-cli commands contain a list or show subcommand, e.g.,
+The cli have two commands that present information to the user: show and list
```
Comment (markdryan) [inline ciao/usage/scripting.md]: The cli has

File: ciao/usage/scripting.md [hunk 2]
```diff
@@ -35,77 +36,72 @@ programmatically retrieve the ssh connection details for the above instance.
...
 }
 ```
 
 ## Template Cheat Sheet
 
 Let's look at a few more examples of how we can use templates to extract
-information from the ciao-cli command.  Looking at the help of the ciao-cli
-instance show command shown above we can see that ciao-cli passes a structure
+information from the ciao command.  Looking at the help of the ciao
+instance show command shown above we can see that ciao passes a structure
```
Comment (markdryan) [inline ciao/usage/scripting.md]: The "ciao show instance" command

**Review by markdryan — COMMENTED** [review]
Just two small comments.

File: ciao/usage/scripting.md (hunk 1 above)
Comment (rbradford) [reply ciao/usage/scripting.md]: Done.

File: ciao/usage/scripting.md (hunk 2 above)
Comment (rbradford) [reply ciao/usage/scripting.md]: Done.

**Review by markdryan — APPROVED** [review]

**Pull request merged**